instrument/config.go

```diff
@@ -96,7 +99,15 @@ func (mc *MetricsConfiguration) NewRootScopeReporter(
 		Prefix:         prefix,
 		CachedReporter: r,
 	}
-	return tally.NewRootScope(scopeOpts, mc.ReportInterval())
+	reportInterval := mc.ReportInterval()
+	scope, closer := tally.NewRootScope(scopeOpts, reportInterval)
+
+	if !mc.DisableRuntimeMetrics {
+		runtimeScope := scope.SubScope("runtime")
```

Comment:
I'm wondering if we want to use a `Tagged` subscope here, e.g. `scope.Tagged(map[string]string{"source": "runtime"})`, to avoid concatenating the names of the subscopes. We've been encouraging other teams to take this approach and it might offer us additional flexibility.